src/commands/data/connectors/update.ts

```diff
@@ -33,12 +33,36 @@ export default class ConnectorsUpdate extends BaseCommand {
         return [key, value]
       },
     }),
+    addTable: flags.string({
```

Comment:
Could we change these to `'add-table'` and `'remove-table'` to match the examples above?

Reply:
Thanks. Changed to addTable, the parser does not like the dashes

Reply:
I believe you can do it with string keys: `'add-table': flags.string`. You'd need to use an alternate syntax when referencing the populated flags: `flags['add-table']`. I don't know if there's a style guide we should be adhering to for this. I don't personally consider it a blocker.